Rewrite the readme to be up-to-date

SysadminJeroen opened the pull request:
A simple update of the README.md file, not much to say about it.

SysadminJeroen committed:
Update the `camosslimageproxy` URLs mentioned.

SysadminJeroen committed:
Update the 'latest' version from 3.1 to 3.2.

SysadminJeroen committed:
Correcting the name of the extension visible in `Manage extensions`.

DavidIQ commented on README.md:
```diff
@@ -3,18 +3,18 @@
 
 ## Quick Install
-You can install this on the latest release of phpBB 3.1 by following the steps below:
+You can install this on the latest release of phpBB 3.2 by following the steps below:
```
Should be "3.1 or 3.2".

Jeroen52 replied:
@DavidIQ committed 25546b3e39c022d2288512ba36ae1ae113a5d3ef.

SysadminJeroen committed:
A reviewer has corrected me to write it down like this.

https://github.com/phpbb-extensions/camosslimageproxy/pull/9#discussion_r102077898
